FCCee/CLD/compact/CLD_o2_v05/materials.xml

```diff
@@ -0,0 +1,240 @@
+    </material>
+
+    <material name="RPCGasDefault" state="gas">
+      <D type="density" value="0.0037" unit="g/cm3"/>
+      <composite n="209" ref="C"/>
+      <composite n="239" ref="H"/>
+      <composite n="381" ref="F"/>
+    </material>
+
+    <material name="Kapton">
+      <D value="1.43" unit="g/cm3" />
+      <composite n="22" ref="C"/>
+      <composite n="10" ref="H" />
+      <composite n="2" ref="N" />
+      <composite n="5" ref="O" />
+    </material>
+
+    <material name="siPCBMix" state="solid">
+      <MEE unit="eV" value="262.475002085268"/>
+      <D unit="g/cm3" value="5.05076923076923"/>
+      <fraction n="0.014498933901919" ref="Cl"/>
+      <fraction n="0.083477099995865" ref="O"/>
+      <fraction n="0.0654857498400853" ref="Si"/>
+      <fraction n="0.00351122019083304" ref="H"/>
+      <fraction n="0.0142636698452849" ref="C"/>
+      <fraction n="0.818763326226013" ref="Cu"/>
+    </material>
+
+    <material name="GroundOrHVMix" state="solid">
+      <MEE unit="eV" value="259.806022507979"/>
+      <D unit="g/cm3" value="5.19"/>
+      <fraction n="0.00360636223506744" ref="H"/>
+      <fraction n="0.0945480597302505" ref="C"/>
+      <fraction n="0.0100234489402697" ref="N"/>
+      <fraction n="0.0286236705202312" ref="O"/>
+      <fraction n="0.863198458574181" ref="Cu"/>
+    </material>
+
+    <material Z="13" name="G4_Al" state="solid">
+      <MEE unit="eV" value="166"/>
+      <D unit="g/cm3" value="2.699"/>
+      <atom unit="g/mole" value="26.9815"/>
+      <fraction n="1" ref="Al"/>
+    </material>
+
+    <material Z="4" name="G4_Be" state="solid">
+      <MEE unit="eV" value="63.7"/>
+      <D unit="g/cm3" value="1.848"/>
+      <atom unit="g/mole" value="9.01218"/>
+      <fraction n="1" ref="Be"/>
+    </material>
+
+    <material name="TungstenDens24">
+      <D value="17.8" unit="g/cm3"/>
+      <fraction n="0.93" ref="W"/>
+      <fraction n="0.061" ref="Ni"/>
+      <fraction n="0.009" ref="Fe"/>
+    </material>
+
+    <material name="PCB" state="solid">
+      <MEE unit="eV" value="88.255598548367"/>
+      <D unit="g/cm3" value="1.7"/>
+      <fraction n="0.180774" ref="Si"/>
+      <fraction n="0.405633" ref="O"/>
+      <fraction n="0.278042" ref="C"/>
+      <fraction n="0.0684428" ref="H"/>
+      <fraction n="0.0671091" ref="Br"/>
+    </material>
+
+    <material name="SolenoidMixture">
+      <D value="4.38" unit="g/cm3"/>
+      <fraction n="0.666" ref="Al"/>
+      <fraction n="0.166" ref="Cu"/>
+      <fraction n="0.084" ref="Ti"/>
+      <fraction n="0.084" ref="Nb"/>
+    </material>
+
+    <material name="AlBeMet162">
+      <D value="2.1" unit="g/cm3"/>
+      <fraction n="0.38" ref="Al"/>
+      <fraction n="0.62" ref="Be"/>
+    </material>
+
+    <!-- this is most likely not it -->
+    <material name="LiquidParaffin">
+      <D value="0.825" unit="g/cm3"/>
+      <composite n="25" ref="C" />
+      <composite n="52" ref="H" />
+    </material>
```

Comment:
```suggestion
    <material name="LiquidParaffin">
      <D value="0.73" unit="g/cm3"/>
      <composite n="10" ref="C" />
      <composite n="22" ref="H" />
    </material>
```

Reply:
Thanks, one also had to modify the average density, or design the beampipe with 3 layers...
(I think actually for the tracking geometry we don't average beampipe materials, because we never considered more than one layer, so I stick with my manual average calculations for now.)